timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -208,54 +226,6 @@ void declare_avg_partial_u_partial_param(py::module &m) {
     });
 }
 
-
-void declare_avg_partial_u_partial_lambda(py::module &m) {
-
-    using Class = timemachine::AvgPartialUPartialLambda;
-    std::string pyclass_name = std::string("AvgPartialUPartialLambda");
-    py::class_<Class, timemachine::Observable>(
-        m,
-        pyclass_name.c_str(),
-        py::buffer_protocol(),
-        py::dynamic_attr()
-    )
-    .def(py::init([](
-        std::vector<timemachine::BoundPotential *> bps,
-        int freq) {
-        return new timemachine::AvgPartialUPartialLambda(bps, freq);
-    }))
-    .def("avg_du_dl", [](timemachine::AvgPartialUPartialLambda &obj) -> double {
-        double avg_du_dl;
-        obj.avg_du_dl(&avg_du_dl);
-        return avg_du_dl;
-    });
-}
-
-
-void declare_full_partial_u_partial_lambda(py::module &m) {
-
-    using Class = timemachine::FullPartialUPartialLambda;
-    std::string pyclass_name = std::string("FullPartialUPartialLambda");
-    py::class_<Class, timemachine::Observable>(
-        m,
-        pyclass_name.c_str(),
-        py::buffer_protocol(),
-        py::dynamic_attr()
-    )
-    .def(py::init([](
-        std::vector<timemachine::BoundPotential *> bps,
-        int freq) {
-        return new timemachine::FullPartialUPartialLambda(bps, freq);
-    }))
-    .def("full_du_dl", [](timemachine::FullPartialUPartialLambda &obj) -> py::array_t<double, py::array::c_style> {
-        int count = obj.count();
-        py::array_t<double, py::array::c_style> py_full_du_dl({count});
-        obj.full_du_dl(py_full_du_dl.mutable_data());
-        return py_full_du_dl;
-    });
-}
-
```

Comment:
Double-check for any usages of `.full_du_dl()` this API change might impact in scripts `dock_and_equilibrate.py`, `rigorous_work.py`

Reply:
Good call - let me see if I can use them. @schmolly we should probably put these three files under testing at some point.

Reply:
Fixed